### ISSUE TITLE
Full HTTP logs for debug builds

### ISSUE THREAD

**Issue by maskaravivek** [commons-app/apps-android-commons#3424]
As of now, the app shows just the basic logs for both debug and release builds. It might be better if we could show the full logs for HTTP builds.

@misaochan If you are okay with it we can make this change.

**Comment by misaochan** [comment]
> if we could show the full logs for HTTP builds.

You mean for debug builds, right? ;)

I am OK with it as long as we take precautions to ensure that the user's password is not displayed like it was the last time we had HTTP logging enabled.

**Comment by maskaravivek** [comment]
Yes, right debug builds.

Redacting the user's password might be difficult. Will it be an issue for debug builds as in most cases the user would be running a debug build using an android studio or his terminal and viewing the logcat. 

If the concern is that he can still accidentally share the logs by going into settings, we can add a warning message specifically for debug builds.

**Comment by misaochan** [comment]
> Redacting the user's password might be difficult. Will it be an issue for debug builds as in most cases the user would be running a debug build using an android studio or his terminal and viewing the logcat.

Yes, I think it could be an issue, it's very possible that a new contributor (or even an old contributor...) could copy paste their debug logs here without checking through the whole thing for sensitive information. IMO it will be a security liability on our part.

Is there a specific need for showing HTTP logs, now that we have fixed the failed uploads issue?
